src/modules/advancedOrders/containers/SettingsWidget/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { useSetAtom } from 'jotai'
+import { useAtomValue } from 'jotai/utils'
+import { useCallback } from 'react'
+
+import { Menu, MenuItem } from '@reach/menu-button'
+
+import { Settings } from 'modules/advancedOrders/pure/Settings'
+import { MenuContent, SettingsButton, SettingsIcon } from 'modules/trade/pure/Settings'
+
+import {
+  advancedOrdersSettingsAtom,
+  updateAdvancedOrdersSettingsAtom,
+  AdvancedOrdersSettingsState,
+} from '../../state/advancedOrdersSettingsAtom'
+
+export function AdvancedOrdersSettings() {
```

Comment:
Can you make the module name match the component?
Either make both SettingsWidget or AdvancedOrdersSettings

Reply:
Makes sense, updated 👍